complete.ly.2.0.js

kofifus opened the pull request:
- code upgraded to ES6 (using Lebab)
- added firstLetterUppercase config option, default is false
- added border config option
- added forceValid option (defaults to true) which will make sure that if the user exits ('enter', 'esc' or 'focusout') the value in the input box will be one of list options
- 'first' and ''options' as config options
- added selectionCallback config option called when selection chages
- list items are not bold
- default input color is black
- show dropdown when partial match and only one item
- no scrollbar when only one item
- narrow scrollbar (webkit only)
- hover changes item background like arrow scroll
- open/hide dropdown on click/lost focus (firefox >=52)
- do not highlight first entry if only one
- hide hint if equal to txtInput value where possible
- removed vertical-align:top (caused problem when embedding in outer divs)
- show dropdown even when only one suggestion
- added maxHeight config option
- make sure elements are scrolled into view when pressing down/up
- all changes in https://github.com/lorenzoongithub/completely/pull/35

Demo in [https://plnkr.co/edit/NPTJmxtOxbKEl6Kg3xrk](https://plnkr.co/edit/NPTJmxtOxbKEl6Kg3xrk)

kofifus committed:
complete.ly.2.0.js

- code upgraded to ES6 (using Lebab)
- added caseSensitive config option, default is false
- list items are not bold
- default input color is black
- show dropdown when partial match and only one item
- no scrollbar when only one item
- narrow scrollbar (webkit only)
- hover change bitem background
- moved options as argument to completely function
- open/hide dropdown on fous/lost focus (firefox >=52)
- do not highlight first entry if only one
- all changes in https://github.com/lorenzoongithub/completely/pull/35

lorenzoongithub commented:
hi kofifus, 
Thanks for your work. I will take a look at it and see if I can integrate it.
heads up:  please be patient it might take me bit (few weeks ?) before I get time.

kofifus committed:
added first and forceValid

completely can now be started with an initial 'first' value

added forceValid option (defaults to true) which will make sure that if the user exits ('ebter', 'esc' or 'focusout') the value in the input box will be one of list options

kofifus committed:
Update complete.ly.2.0.js

kofifus committed:
Update complete.ly.2.0.js

kofifus committed:
Update complete.ly.2.0.js

kofifus committed:
Update complete.ly.2.0.js

kofifus committed:
Update complete.ly.2.0.js

kofifus committed:
Update complete.ly.2.0.js
